src/azul/service/repository.py

```diff
@@ -84,12 +84,23 @@ def get_summary(self, filters):
         # Request a summary for each entity type and cherry-pick summary fields from the summaries for the entity
         # that is authoritative for those fields.
         summary_fields_by_authority = {
-            'files':
-                ['totalFileSize', 'fileTypeSummaries', 'fileCount'],
-            'samples':
-                ['organCount', 'donorCount', 'labCount', 'totalCellCount', 'organSummaries', 'specimenCount'],
-            'projects':
-                ['projectCount']
+            'files': [
+                'totalFileSize',
+                'fileTypeSummaries',
+                'fileCount',
+            ],
+            'samples': [
+                'organTypes',
+                'donorCount',
+                'specimenCount',
+            ],
+            'projects': [
+                'projectCount',
+                'labCount',
+                # Get cell count from projects summary to avoid count duplication in the case of multiple samples
```

Comment:
Could be better. "in the case of multiple samples" is not very precise.

Reply:
Expanded on explanation in the comment.